src/igvfd/schemas/file.json

```diff
@@ -31,20 +31,28 @@
     ],
     "dependentSchemas": {
         "derived_manually": {
-            "comment": "Specification of derived_manually is mutually exclusive with specification of analysis_step_version.",
-            "not": {
-                "required": [
-                    "analysis_step_version"
-                ]
-            }
-        },
-        "analysis_step_version": {
-            "comment": "Specification of analysis_step_version is mutually exclusive with specification of derived_manually.",
-            "not": {
-                "required": [
-                    "derived_manually"
-                ]
-            }
+            "comment": "If derived_manually is true, analysis_step_version is not required.",
+            "oneOf": [
+                {
+                    "properties": {
+                        "derived_manually": {
+                            "const": true
+                        }
+                    },
+                    "not": {
+                        "required": [
```

Comment:
is it intended here if dervied_manually = true, then you can't link analysis_step_version to the file?

Reply:
yes, since it is derived_manually, no softwares/softare version/analysis step are used so no analysis step version is needed.